CANARY.md

```diff
@@ -0,0 +1,13 @@
+# Canary strat
```

Comment:
i don't really understand why this doc is here, in the root of the ob repo

or who it is for?

@rouzwelt originally i asked you to setup a canary, which is already documented in the strat in pubstrats in basically paraphrasing exactly what is here

```

# The canary order does nothing except run on a cooldown.
#
# It is useful only to show that orders clear on some network. If the canary is
# deployed and does not run, it means that there are no solvers monitoring the
# network, or at least not the raindex contract that the canary is deployed on.
#
# The canary is basically a stripped back version of the sampler. It runs but
# doesn't sample anything.
```

i'd expect to see docs that explain what was missing in your understanding when i asked you about it @rouzwelt and probably somewhere related to the infra they are for